pypsa/plot.py

```diff
@@ -254,15 +253,95 @@ def as_branch_series(ser):
     ax.update_datalim(compute_bbox_with_margins(margin, x, y))
     ax.autoscale_view()
 
+    if basemap_present:
+        ax.axis('off')
+    if cartopy_present and drawmap:
+        ax.outline_patch.set_visible(False)
+
     ax.set_title(title)
 
     return (bus_collection,) + tuple(branch_collections)
 
 
+def compute_bbox_with_margins(margin, x, y):
+    'Helper function to compute bounding box for the plot'
+    # set margins
+    pos = np.asarray((x, y))
+    minxy, maxxy = pos.min(axis=1), pos.max(axis=1)
+    xy1 = minxy - margin*(maxxy - minxy)
+    xy2 = maxxy + margin*(maxxy - minxy)
+    return tuple(xy1), tuple(xy2)
+
+
+def projected_area_factor(ax):
+    """
+    Helper function to get the area scale of the current projection in
+    reference to the default projection.
+    """
+    if not 'projection' in ax.__dict__:
+        return 1
+    if ax.projection is ccrs.PlateCarree():
+        return 1
+    x1, x2, y1, y2 = ax.get_extent()
+    pbounds = \
+        ccrs.PlateCarree().transform_points(ax.projection,
```

Comment:
Actually this is fine, as the points should be transformed to the original projection (which was assumed to be PlateCarree, however should be given in srid). Ideally this would achieved by 

```python 
    pbounds =  ccrs.epsg(net.srid).transform_points(ax.projection, 
                                np.array([x1, x2]), np.array([y1, y2]))
```

which is  however not possible as the default net.srid is not a valid epsg code (see comment above)

Reply:
Correct. Only see above.